src/frontend/src/lib/components/icons/IconWallet.svelte

```diff
@@ -0,0 +1,25 @@
+<!-- source: DFINITY foundation -->
```

Comment:
Different that the gix-cmp `IconWallet`?

Reply:
Yep, this is double color